system/reports/assets/simple.cfm

```diff
@@ -309,32 +311,52 @@ code {
 	<cfreturn bootstrapClass>
 </cffunction>
 
+<cffunction name="statusToEmoji" output="false">
+	<cfargument name="status">
+	<cfargument name="emojiService">
```

Comment:
Why pass it, the template has access to it already

Reply:
I found this while researching, but in our case, the functions are in the functions local scope, they are `within` the function. I might be a bug. https://stackoverflow.com/questions/41749954/in-coldfusion-udfs-is-there-a-way-to-reference-a-variable-from-the-parent-functi

Reply:
It is an issue with nested functions, because outside of them `emojiService` IS accessible.